nginx-directory/files/index.html

```diff
@@ -1,20 +1,122 @@
 <html>
-  <head>
-    <style>
+
+<head>
+  <style>
+    *,
+    *:before,
+    *:after {
+      box-sizing: border-box;
+    }
+
+    body {
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
+      margin: 0;
+      padding: 24px;
+      color: #424242;
+      line-height: 1.7;
+      background: #fff;
+    }
+
+    h1 {
+      border-bottom: 1px solid #e4e4e4;
+    }
+
+    .help {
+      color: #666;
+      text-decoration-line: underline;
+      text-decoration-style: dashed;
+      vertical-align: super;
+    }
+
+    ul {
+      list-style: none;
+      margin: 0;
+      padding: 0;
+    }
+
+    li {
+      margin: 0;
+      padding: 0;
+    }
+
+    a {
+      display: inline-block;
+      padding: 8px;
+      color: #424242;
+      text-decoration: none;
+    }
+
+    a:hover {
+      background: #f1f5f9;
+      color: #3865AB;
+    }
+
+    table {
+      border-collapse: collapse;
+    }
+
+    th,
+    td {
+      border: 1px solid #e4e4e4;
+      padding: 8px 12px;
+      text-align: left;
+    }
+
+    th {
+      background: #f8f9fa;
+      font-weight: 500;
+    }
+
+    button,
+    input[type="button"] {
+      padding: 4px 12px;
+    }
+
+    @media (prefers-color-scheme: dark) {
```

Comment:
as I understand this, it will always sync with the "system" setting, right?

no opinion on that right now - just wanted to call it out and see what others think. might be that if we did this we'd like a toggle that saves the preference in local storage, because not all folks will like that.